puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/nightly_packaging.groovy

```diff
@@ -0,0 +1,30 @@
+def setup_nightly_build_environment(args) {
+    def commit_hash = ''
+
+    dir(args.package_name) {
+        git(url: "https://github.com/${args.github_repo}.git", branch: 'develop')
+        commit_hash = archive_git_hash()
+    }
+    dir('foreman-packaging') { git(url: 'https://github.com/theforeman/foreman-packaging.git', branch: 'rpm/develop') }
```

Comment:
linebreaks, please :)

Reply:
I thought this looked cleaner in-line being that it was just a single `git()` call, but I'm okay wit uniformity :)